content/shell/android/shell_library_loader.cc

```diff
@@ -7,8 +7,11 @@
 #include "content/public/app/content_main.h"
 #include "content/shell/app/shell_main_delegate.h"
 
+#include "starboard/system.h"
+
 // This is called by the VM when the shared library is first loaded.
 JNI_EXPORT jint JNI_OnLoad(JavaVM* vm, void* reserved) {
+  SbSystemGetTotalCPUMemory();
```

Comment:
DNS

Reply:
Thanks for testing it works ! :+1: 

@xiaomings here you go for a a demo !